EIPS/eip-3664.md

```diff
@@ -0,0 +1,308 @@
+---
+eip: 3664
+title: Next generation game NFT
+author: TylerZ (@tylerztl), TY (@nimocat)
+discussions-to: https://ethereum-magicians.org/t/eip-3664-next-generation-game-nft-standard/6738
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-17
+requires: 20, 1155
+---
+
+## Simple Summary
```

Comment:
Hi, this field was recently deprecated in favor of a `description` field in the preamble. Please see the [eip template](./eip-template.md) for proper formatting. Thanks!

Reply:
Thanks lightclient, already updated. Also updated the motivation. Thanks for your guide.

Reply:
@lightclient if you have any suggestions on our EIP, just tell us and we will make efforts on every aspects for pushing forward this EIP draft.